tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +95,26 @@ java -cp {cp} {name} "$@"
       output=ctx.outputs.executable,
       content=content)
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
+  return args
+
+def _write_test_launcher(ctx, jars):
+  content = """#!/bin/bash
+cd $0.runfiles
+java -cp {cp} {name} {args} "$@"
```

Comment:
Don't depends on java being on the PATH, instead add dependency to //tools/defaults:jdk and to @bazel_tools//tools/jdk:java and use the latter as your java binary here.